### ISSUE TITLE
[Bug?]: netdata and fireqos export to prometheus 

### ISSUE THREAD

**Issue by mvivaldi** [netdata/netdata#15261]
### Bug description

I have netdata with fireqos and everything is working fine, but prometheus is unable to collect the data from /api/v1/allmetrics with this error:
```
label name "family" is not unique: invalid sample
```
in the metrics exported by netdata I found this:
```
......
netdata_tc_qos_kilobits_persec_average{chart="tc.world-out",dimension="k8s",family="enp3s0",device="enp3s0",name="world-out",family="enp3s0"} 3.9788264 1687868630000
netdata_tc_qos_kilobits_persec_average{chart="tc.world-out",dimension="wireguard",family="enp3s0",device="enp3s0",name="world-out",family="enp3s0"} 253.7405038 1687868630000
netdata_tc_qos_kilobits_persec_average{chart="tc.world-out",dimension="server",family="enp3s0",device="enp3s0",name="world-out",family="enp3s0"} 0.8074255 1687868630000
netdata_tc_qos_kilobits_persec_average{chart="tc.world-out",dimension="synacks",family="enp3s0",device="enp3s0",name="world-out",family="enp3s0"} 10.6554646 1687868630000
netdata_tc_qos_kilobits_persec_average{chart="tc.world-out",dimension="nginx",family="enp3s0",device="enp3s0",name="world-out",family="enp3s0"} 4.0004681 1687868630000
......
```
in every netdata_tc_qos_* metrics I have the label family duplicated

I don't know if this is a bug of netdata (1.40.0) fireqos (3.1.7) or maybe something has changed in prometheus (2.45.0)

### Expected behavior

the metrics in /api/v1/allmetrics shouldn't have duplicated labels

### Steps to reproduce

1. install netdata and fireqos
2. query the metrics and verify if the label family is duplicated in the  netdata_tc_qos_* metrics

### Installation method

manual setup of official DEB/RPM packages

### System info

```shell
# uname -a; grep -HvE "^#|URL" /etc/*release
Linux xenserver 5.19.0-43-generic #44~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Mon May 22 13:39:36 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
/etc/lsb-release:DISTRIB_ID=Ubuntu
/etc/lsb-release:DISTRIB_RELEASE=22.04
/etc/lsb-release:DISTRIB_CODENAME=jammy
/etc/lsb-release:DISTRIB_DESCRIPTION="Ubuntu 22.04.2 LTS"
/etc/os-release:PRETTY_NAME="Ubuntu 22.04.2 LTS"
/etc/os-release:NAME="Ubuntu"
/etc/os-release:VERSION_ID="22.04"
/etc/os-release:VERSION="22.04.2 LTS (Jammy Jellyfish)"
/etc/os-release:VERSION_CODENAME=jammy
/etc/os-release:ID=ubuntu
/etc/os-release:ID_LIKE=debian
/etc/os-release:UBUNTU_CODENAME=jammy
```

### Netdata build info

```shell
# netdata -W buildinfo
Version: netdata v1.40.0
Configure options:  '--build=x86_64-linux-gnu' '--includedir=${prefix}/include' '--mandir=${prefix}/share/man' '--infodir=${prefix}/share/info' '--disable-option-checking' '--disable-si
lent-rules' '--libdir=${prefix}/lib/x86_64-linux-gnu' '--libexecdir=${prefix}/lib/x86_64-linux-gnu' '--disable-maintainer-mode' '--prefix=/usr' '--sysconfdir=/etc' '--localstatedir=/var
' '--libdir=/usr/lib' '--libexecdir=/usr/libexec' '--with-user=netdata' '--with-math' '--with-zlib' '--with-webdir=/var/lib/netdata/www' '--disable-dependency-tracking' 'build_alias=x86
_64-linux-gnu' 'CFLAGS=-g -O2 -ffile-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security' 'LDFLAGS=-Wl,-Bsymbolic-functions -Wl,-z,relro' 'CPPFLAGS=-
Wdate-time -D_FORTIFY_SOURCE=2' 'CXXFLAGS=-g -O2 -ffile-prefix-map=/usr/src/netdata=. -fstack-protector-strong -Wformat -Werror=format-security'
Install type: binpkg-deb
    Binary architecture: x86_64
    Packaging distro:  
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES 
    ACLK:                       YES
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         YES
    HTTPD (h2o):                YES
Libraries:
    protobuf:                YES (system)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    YES
    debugfs:                 YES
    EBPF:                    YES
    IPMI:                    YES
    NFACCT:                  YES
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: YES
Debug/Developer Features:
    Trace Allocations:       NO
```

**Comment by ilyam8** [comment]
Thanks for reporting, @mvivaldi. Fixed in #15262.

**Comment by ilyam8** [comment]
@mvivaldi the fix is merged, but (unfortunately) it didn't make it into the patch release for some reason. The workaround is to filter out all tc plugin metrics. To do it:

 - open exporting.conf
 - uncomment and set `send charts matching` to `!tc* *` (`[prometheus:exporter]` section)
 - restart netdata service
